packages/server/graphql/private/mutations/verifyDomain.ts

```diff
@@ -0,0 +1,32 @@
+import getRethink from '../../../database/rethinkDriver'
+import {MutationResolvers} from '../resolverTypes'
+import normalizeSlugName from './helpers/normalizeSlugName'
+
+const verifyDomain: MutationResolvers['verifyDomain'] = async (_source, {slug, domains, orgId}) => {
+  const r = await getRethink()
+  const normalizedDomains = domains.map((domain) => domain.toLowerCase())
+
+  // VALIDATION
+  const slugName = normalizeSlugName(slug)
+  if (slugName instanceof Error) return {error: {message: slugName.message}}
+
+  const [slugNameExist, orgIdExist] = await Promise.all([
+    r.table('SAML')('id').count(slugName).eq(1).run(),
+    r.table('SAML')('orgId').count(orgId).eq(1).run()
```

Comment:
+1 I noticed that the queries in this form this do not use indexes

We can write something like this instead:

```
await Promise.all([
  r.table('SAML').get(slugName).run(),     
    r.table('SAML').getAll(orgId, {index: 'orgId'}).limit(1)
    .count()
    .eq(1).run()
])